api/src/index.ts

```diff
@@ -0,0 +1,159 @@
+import * as didSdk from '@adxp/did-sdk'
+import * as nameSdk from './names.js'
+import * as ucan from 'ucans'
+import { Keypair } from '../common/types.js'
+import * as t from './types.js'
+
+export class AdxClient {
```

Comment:
Does AdxClient need to be a class? ie will it encapsulate some state? Otherwise maybe we just make it functions. that way when calling, you don't have to instantiate every time
```ts
const client = new AdxClient()
client.someMethod(...)
```